yarn.lock

```diff
@@ -7713,10 +7713,10 @@ vanilla-framework@2.19.1:
   resolved "https://registry.yarnpkg.com/vanilla-framework/-/vanilla-framework-2.19.1.tgz#f071deed5504a5fe2104d6d9b087ce654e190881"
   integrity sha512-y79SKAwertvgHVAf1PbGxiq6QvVxNI78txilSgESx80HUWBrooT1nXCDUuHC/ySGEq+JUpmVIv0PrmXXilYWdg==
 
-vanilla-framework@2.21.0:
-  version "2.21.0"
-  resolved "https://registry.yarnpkg.com/vanilla-framework/-/vanilla-framework-2.21.0.tgz#9d0b9b7cabd2a8cd4ccf452c471445e611868d69"
-  integrity sha512-KIX/hBl3b7vN3B0HDCj+1NadDUa6WtOEC0HTHNVvyQ2yC+pvQN2WHA1yVbUn7KaGLdv0RBqfNm88YlQ1wRPtbA==
+vanilla-framework@2.22.0:
```

Comment:
package.json is updated to 2.22.1, but yarn.lock is not

Reply:
fixed it.